support-frontend/assets/helpers/urls/routes.ts

```diff
@@ -134,6 +134,26 @@ function paperCheckoutUrl(
 	return addQueryParamsToURL(url, params);
 }
 
+function parameteriseUrl(
```

Comment:
Would it be worth adding some tests to this to verify the behaviour and act as documentation?